README.md

```diff
@@ -96,6 +96,55 @@ class EmailHandler(tornado.web.RequestHandler, PostmarkMixin):
         subject = 'Test Message'
         self.send_email(body=body, to=to, subject=subject)
 ```
+
+Postmark Templates
+------------------
+Getting an HTML email to look good on most email clients can be tricky. The whole reason to send email as HTML is to make an impression on the reader. If the client renders it badly, its worse than just sending a plain text email.
+
+Postmark understands this and has created a templating system to make it easier to create robust HTML emails. For more information, see: [Special delivery: Postmark templates] (https://postmarkapp.com/blog/special-delivery-postmark-templates)
+
+Using templates in python-postmark is straightforward: 
+
+1. Create a template for your server on the Postmark website.
+
+2. Get the ID for the template from the page on Postmark where the template was made.
+
+3. Make a dict in python that contains the values of the template variables. Postmark calls this the "TemplateModel"
+
+4. Instantiate PMMail with the the appropriate kwargs, including "template_id" and "template_model". NOTE: do not set the "subject" kwarg.
+
+5. Call the send_with_template() method
+
+Here is an example:
+
+```
+from postmark.core import PMMail
+
+template_model = {
+    'firm_name':'Acme Law',
+    'total': '125.43',
+    'date': 'July 10, 2016',
+    'client_name': 'Joe Blow',
+    'invoice_details': [
+        {
+            'description': 'Credit report',
+            'amount': '25.43'
+        },
+        {
+            'description': 'Legal Services',
+            'amount': '100.00'
+        }
+    ],
+    'opening_balance': '2000.00',
+    'closing_balance': "<span class='negative_amount'>1800.00</span>",
+    'notes': 'Thank you for choosing Acme Law.',
+}
+pm = PMMail(to='groucho.marx@gmail.com', sender='harpo.marx@gmail.com',
+            template_id=123456, template_model=template_context)
```

Comment:
Where is template_context coming from? Shouldn't it be `template_model=template_model`?